File: JAVASCRIPT/Breakfast/javascript/countPrimeNumber.js
let number = 0
let count = 0
let count1 = 0
while number<100:
	number+=1
	if (number == 1 || number == 3 || number == 5 || number == 7 || number == 11 || number == 13){
		count+=1
	}
	if (number % 2 !=0 ){
		count1+=1
	}
console.log(count+count1)

File: JAVASCRIPT/Breakfast/Java/CountPrimeNumber.java
public class CountPrimeNumber{
public static void main(String[]args){

int number = 0;
int count = 0;
int count1 = 0;
while (number<100){
	number+=1;
	if (number == 1 || number == 3 || number == 5 || number == 7 || number == 11 || number == 13){
		count+=1;
	}
	if (number % 2 !=0 ){
		count1+=1;
	}
}
System.out.print(count+count1);

}
}